src/server/iframe.html.js

```diff
@@ -3,6 +3,11 @@ export default function (headHtml) {
     <!DOCTYPE html>
     <html>
       <head>
+        <script>
+          if (window.parent !== window) {
```

Comment:
I think that we can safely assume this will always be served from the same origin and the storybook, and we don't need to worry about that. In fact, we can also remove the `window.parent !== window` check